app/models/wpcc/percent.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  class Percent
+    def self.choose_highest(period_percent, user_percent)
```

Comment:
I still think Percent is enough since we don't have any other type of percent on the tool. And the utility will not bring anything new to the class. I don't want to be stubborn but Percent has a rule to prioritize user percent over period percent and that's the reason that IMHO the variable period percent it is the better name.